lib/tasks/images.js

```diff
@@ -85,11 +85,18 @@ module.exports = function(gulp, config) {
 			var transformStream = new stream.Transform({objectMode: true});
 			transformStream._transform = function (file, encoding, callback) {
 				if(file.imageTransform) {
-					sharp(file.contents).resize(file.imageTransform.width, file.imageTransform.height).withoutEnlargement().jpeg(file.imageTransform.options).toBuffer(function (err, buffer, info) {
-						if(err) console.error(err)
-						file.contents = buffer
-						callback(null, file)
-					})
+					jimp
+						.read(file.contents)
+						.then(function(image) {
+							image
+								.resize(file.imageTransform.width, file.imageTransform.height)
+								.quality(image.quality)
+								.getBuffer(jimp.AUTO, function(err, buffer) {
+									if(err) console.error(err)
```

Comment:
Was so in sharp version as well, but: Shouldn't we pass the error up in the callback? Not just silently dismiss it?

Reply:
That's the question - is an broken image a fatal fail or just a inconvience?

Reply:
If the image is then missing it's a fail.